core/src/main/java/com/orange/ouds/core/component/OudsNavigationBar.kt

```diff
@@ -0,0 +1,511 @@
+ * @param icon Icon of the item.
+ * @param modifier [Modifier] applied to the navigation bar item.
+ * @param label Label of the item.
+ * @param badge Optional badge display on the item icon.
+ * @param alwaysShowLabel Whether the label should always be shown.
+ * @param interactionSource [MutableInteractionSource] that will be used to dispatch events when this item is pressed, hovered or focused.
+ *
+ * @sample com.orange.ouds.core.component.samples.OudsNavigationBarSample
+ */
+@Composable
+fun RowScope.OudsNavigationBarItem(
+    selected: Boolean,
+    onClick: () -> Unit,
+    icon: OudsNavigationBarItemIcon,
+    modifier: Modifier = Modifier,
+    label: String? = null,
+    badge: OudsNavigationBarItemBadge? = null,
+    alwaysShowLabel: Boolean = true,
+    interactionSource: MutableInteractionSource? = null
+) {
+    @Suppress("NAME_SHADOWING") val interactionSource = interactionSource ?: remember { MutableInteractionSource() }
+    val interactionState by interactionSource.collectInteractionStateAsState()
+    val state = getNavigationBarItemState(interactionState = interactionState)
+    with(OudsTheme.componentsTokens.bar) {
+        val selectedContentColor = contentColor(state = state, selected = true)
+        val unselectedContentColor = contentColor(state = state, selected = false)
+
+        ConstraintLayout(
+            modifier = modifier
+                .selectable(
+                    selected = selected,
+                    onClick = { },
+                    role = Role.Tab
+                )
+        ) {
+            val (itemRef, topIndicatorRef) = createRefs()
+
+            // Top active indicator: visual alternative for selected item (a11y)
+            val topIndicatorColor = topIndicatorColor(state = state)
+            val topIndicatorOpacityColor = topIndicatorColor.copy(alpha = topIndicatorColor.alpha * opacityActiveIndicatorCustom.value)
+            val topIndicatorShape = RoundedCornerShape(
+                topStart = 0.dp,
+                topEnd = 0.dp,
+                bottomStart = borderRadiusActiveIndicatorCustomTop.value,
+                bottomEnd = borderRadiusActiveIndicatorCustomTop.value
+            )
+            this@OudsNavigationBarItem.AnimatedVisibility(
+                modifier = Modifier.constrainAs(topIndicatorRef) {
+                    top.linkTo(parent.top)
+                    start.linkTo(parent.start)
+                    end.linkTo(parent.end)
+                    height = Dimension.value(sizeHeightActiveIndicatorCustom.dp)
+                    width = Dimension.value(sizeWidthActiveIndicatorCustomTop.dp)
+                },
+                visible = selected || state == OudsNavigationBarItemState.Hovered,
+                enter = fadeIn() + slideInVertically { -it * 2 },
+                exit = fadeOut() + slideOutVertically { -it * 2 }
+            ) {
+                Box(modifier = Modifier.background(color = topIndicatorOpacityColor, shape = topIndicatorShape))
```

Comment:
That's equivalent to applying the alpha to the color because there is no content in the indicator, but we can also apply this opacity direcly using the `alpha` modifier on the box:

```kotlin
modifier = Modifier
    .background(color = topIndicatorColor, shape = topIndicatorShape)
    .alpha(opacityActiveIndicatorCustom.value)
```